src/ansible/roles/selenium/files/idp_login_keycloak.py

```diff
@@ -0,0 +1,45 @@
+#!/opt/test_venv/bin/python3
+
+import sys
+
+from selenium import webdriver
+from datetime import datetime
+from packaging.version import parse
+from selenium.webdriver.firefox.options import Options
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
```

Comment:
Let's add input validation before actually using the values. I know this is a test script but that would add an additional robustness to it. Just checking that the number of args is the expected one should be enough.

Reply:
Perfect